face/faces/ImageFace.js

```diff
@@ -0,0 +1,60 @@
+//
+//  BlockV AG. Copyright (c) 2018, all rights reserved.
+//
+//  Licensed under the BlockV SDK License (the "License"); you may not use this file or
+//  the BlockV SDK except in compliance with the License accompanying it. Unless
+//  required by applicable law or agreed to in writing, the BlockV SDK distributed under
+//  the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
+//  ANY KIND, either express or implied. See the License for the specific language
+//  governing permissions and limitations under the License.
+//
+import BaseFace from './BaseFace';
+
+export default class ImageFace extends BaseFace {
+  // eslint-disable-next-line
+  constructor(vatomView, vatom, face) {
+    super(vatomView, vatom, face);
+    this.onLoad();
```

Comment:
VatomView is calling onLoad(), it shouldn't be called here, it'll end up loading twice...